test/e2e/exec_test.go

```diff
@@ -17,13 +17,7 @@ var _ = Describe("Podman exec", func() {
 	It("podman exec into bogus container", func() {
 		session := podmanTest.Podman([]string{"exec", "foobar", "ls"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(125))
-	})
-
-	It("podman exec without command", func() {
-		session := podmanTest.Podman([]string{"exec", "foobar"})
-		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(125))
+		Expect(session).Should(ExitWithError(125, "no container with name or ID \"foobar\" found: no such container"))
```

Comment:
This is why I'm "wasting" my time on this seemingly-trivial exercise.

This entire test was pointless. I'm sure the author wanted it to fail with "must provide a non-empty command", but in reality it was failing with "container foobar does not exist and I'm not even going to bother checking the rest of the args".

I added this test (as intended) as a piggyback onto `simple command` below.